packages/grafana-ui/src/components/Slider/Slider.tsx

```diff
@@ -99,15 +129,14 @@ export const Slider: FunctionComponent<SliderProps> = ({
           marks={marks}
           included={included}
         />
-        {/* Uses text input so that the number spinners are not shown */}
+        {/* Uses text input so that the number spinners are not shown; manually implements arrow keys and validation */}
```

Comment:
@dprokop -- the changes to `<Slider .../>` are needed because the current input field allows you to type values that are invalid in the slider!

Do you think it is OK to do the changes here, or should we make a new PR?  This issue became clear because the slider allows 2-128 steps, and is most interesting in the low numbers.

Reply:
@ryantxu I would suggest extracting those changes to a separate PR as it's a change to the public API of a component

Not sure if the plugins platform squad has already established any rules regarding that(cc @mckn), but it would be nice to see Levitate highlight these kinds of changes as they occur - I think levitate did that at the beginning, didn't it @leventebalogh ?